fuel-core-interfaces/src/db.rs

```diff
@@ -792,33 +792,33 @@ pub mod helpers {
     }
 
     // Validator set. Used by relayer.
-    impl Storage<Address, (ValidatorStake, Option<Address>)> for DummyDb {
+    impl Storage<ValidatorId, (ValidatorStake, Option<ConsensusId>)> for DummyDb {
         type Error = crate::db::KvStoreError;
 
         fn insert(
             &mut self,
-            key: &Address,
-            value: &(ValidatorStake, Option<Address>),
-        ) -> Result<Option<(ValidatorStake, Option<Address>)>, Self::Error> {
+            key: &ValidatorId,
+            value: &(ValidatorStake, Option<ConsensusId>),
+        ) -> Result<Option<(ValidatorStake, Option<ConsensusId>)>, Self::Error> {
             Ok(self.data.lock().validators.insert(*key, *value))
         }
 
         fn remove(
             &mut self,
-            _key: &Address,
-        ) -> Result<Option<(ValidatorStake, Option<Address>)>, Self::Error> {
+            _key: &ValidatorId,
+        ) -> Result<Option<(ValidatorStake, Option<ConsensusId>)>, Self::Error> {
             unreachable!()
         }
 
         fn get<'a>(
             &'a self,
-            key: &Address,
-        ) -> Result<Option<std::borrow::Cow<'a, (ValidatorStake, Option<Address>)>>, Self::Error>
+            key: &ValidatorId,
+        ) -> Result<Option<std::borrow::Cow<'a, (ValidatorStake, Option<ConsensusId>)>>, Self::Error>
         {
             Ok(self.data.lock().validators.get(key).map(|i| Cow::Owned(*i)))
         }
 
-        fn contains_key(&self, _key: &Address) -> Result<bool, Self::Error> {
+        fn contains_key(&self, _key: &ValidatorId) -> Result<bool, Self::Error> {
             unreachable!()
```

Comment:
Ditto; out of context `unreachable`

Reply:
Task for it: https://github.com/FuelLabs/fuel-core/issues/431